docs/lifetime/startup.rst

```diff
@@ -265,6 +265,56 @@ You'll see from the output that the callbacks and ``OnActivated`` methods execut
 
 Note if you don't use ``SingleInstance`` then ``OnActivated`` will be called for *every new instance of the dependency*. Since "warm start" objects are usually singletons and are expensive to create, this is generally what you want anyway.
 
+Lifetime Scopes
+===============
+
+Registering an ``IStartable`` or ``AutoActivate`` item with something other than ``SingleInstance`` or ``InstancePerDependency`` may not work the way you expect.
+
+For example, if you register using ``InstancePerLifetimeScope``, **this does not result in a new startable running in every lifetime scope you create**. The startable will instead run on container build only.
+
+Further, **you can't use IStartable or AutoActivate with named lifetime scopes.** Registering with a named lifetime scope won't start the component when the named scope is created; instead it will yield an exception on container build because the named scope doesn't exist.
+
+.. sourcecode:: csharp
+
+    static void Main(string[] args)
+    {
+        var builder = new ContainerBuilder();
+
+        // This WON'T WORK. You'll get a DependencyResolutionException when
+        // the container tries to start the component because the named lifetime
+        // scope doesn't exist.
+        builder.RegisterType<Startable1>()
+               .As<IStartable>()
+               .InstancePerMatchingLifetimeScope("unitofwork");
+        builder.Build();
+    }
+
+If you need to start something in a particular lifetime scope, you need to register it with that lifetime scope at the time of creation.
```

Comment:
Perhaps you should say "at the time of creating the scope", for extra clarity?

Reply:
Updated to:

> If you need to start something in a particular lifetime scope, you need to register it with that lifetime scope at you create the scope (i.e., in the ``BeginLifetimeScope`` call).